packages/terra-application-utility/src/utility/_UtilityMenuDivider.module.scss

```diff
@@ -5,6 +5,7 @@
     border-right-width: var(--terra-application-utility-menu-divider-border-bottom-right, 0);
     border-top: var(--terra-application-utility-menu-divider-border-bottom, 1px solid #dedfe0);
     height: var(--terra-application-utility-menu-divider-height, 0.2143rem);
+    outline: none;
```

Comment:
If the diver should never receive focus removing the tabIndex attribute would also prevent tab and click focus entirely, assuming the tabIndex attribute isn't playing another role.

> Because tabindex="-1" makes elements non-navigable via the keyboard, make sure you don’t include it on any form, object, or link elements a user could click on with a mouse or receive focus via touch.

https://www.wufoo.com/html5/tabindex-attribute/

Reply:
ohh i didn't know that. that's a better solution - updated